test/versioned-external/external-repos.js

```diff
@@ -19,6 +19,7 @@ const repos = [
     additionalFiles: [
       'tests/agent-testing.js',
       'tests/create-apollo-server-setup.js',
+      'tests/custom-assertions.js',
```

Comment:
i think long term we should prob move all these files into a `tests/lib` and then update this stanza to just include that folder